app/cisc/cisc.go

```diff
@@ -76,9 +80,11 @@ func idCreate(c *cli.Context) error {
 	}
 	log.Info("Creating new blockchain-identity for", name)
 
-	cfg := identity.NewIdentity(group.Roster, 2, name)
+	thr := c.Int("thr")
```

Comment:
You use `majority` and `threshold` interchangeably. Can we make this more consistent? In `NewIdentity` call the param threshold (then the above is also clear).